passport-reshuffle/src/auth-handler.ts

```diff
@@ -0,0 +1,141 @@
+import { makeStrategy, isFake } from './strategy';
```

Comment:
I think the fake functions should live in a different file to make it clear they are only for the local dev use case
(not for this PR)